store/build.gradle

```diff
@@ -11,3 +14,25 @@ kotlin {
     coroutines 'enable'
   }
 }
+
+sqldelight {
+  packageName = "com.jakewharton.sdksearch.store"
+}
+
+compileKotlinCommon.dependsOn("generateSqlDelightInterface")
+
+afterEvaluate {
+  project.tasks.withType(Test) { task ->
+    task.dependsOn("verifySqlDelightMigration")
```

Comment:
I think this should be `tasks.getByName("check").dependsOn("verifySqlDelightMigration")`.

Should the plugin do this?

Reply:
i wasnt sure if we should force the sqldelight migration strategy on consumers

Reply:
Does it no-op when there aren't any .sqm files?

Reply:
it no-ops when there are no .db files